wetmap/src/components/newModals/picUploader/view.tsx

```diff
@@ -13,6 +13,7 @@ import WavyModalHeader from '../../reusables/wavyModalHeader';
 import style from './style.module.scss';
```

Comment:
it's not showing here but it looks like useRef is being imported but not used, so you can delete it from the import I think

Reply:
Right. I've missed that. Thanks.